addons/account/views/report_invoice.xml

```diff
@@ -113,7 +114,7 @@
                                     <th name="th_discount" t-if="display_discount" t-attf-class="text-end {{ 'd-none d-md-table-cell' if report_type == 'html' else '' }}">
                                         <span>Disc.%</span>
                                     </th>
-                                    <th name="th_taxes" t-attf-class="text-start {{ 'd-none d-md-table-cell' if report_type == 'html' else '' }}"><span>Taxes</span></th>
+                                    <th name="th_taxes" t-attf-class="text-start" t-att-style="'display: none; !important;' if not show_taxes else ''"><span>Taxes</span></th>
```

Comment:
Avoid doing inline style, it's a bad practice, we should always try to use css class from bootstrap or custom classes prefixed by o_....

Why do we avoid using inline style:

Maintainability: Harder to update and manage as the codebase grows.
Readability: Combining HTML and styling reduces code clarity.
Reusability: Lack of reusability may lead to redundant code.
Limited Control: Restricts the use of advanced CSS features. Because it has an higher priority than css classes.
Cascading Issues: Overrides external styles, causing unexpected behavior.